.changeset/warm-hotels-allow.md

```diff
@@ -0,0 +1,114 @@
+---
+"@shopify/shopify-api": minor
+---
+Updates the Session class to handle the associated user information on the session object.
+
+Updates the Session `fromPropertyArray` to handle all user info fields. New optional argument `returnUserData`, (defaulted to`false`), will return the user data as part of the session object.
+
+```ts
```

Comment:
I wonder if we want to put these examples inside a `<details>` element so it doesn't make the changelog too long. Although that makes me worry whether it will be visible enough.

Reply:
I think I will put it in a details element.